crates/node/src/main.rs

```diff
@@ -296,9 +310,13 @@ async fn run(config: Arc<Config>) -> Result<()> {
     .await?;
 
     tracing::info!("gevulot node started");
-    loop {
-        sleep(Duration::from_secs(1));
+    // loop {
+    //     sleep(Duration::from_secs(1));
+    // }
+    if let Err(err) = download_jh.await {
+        tracing::info!("dwn mng error:{err}");
```

Comment:
```suggestion
        tracing::info!("download_manager error:{err}");
```
Less confusing if suddenly confronted with this one :sweat_smile: